src/features/defi/providers/cosmos/components/CosmosManager/Claim/components/Confirm.tsx

```diff
@@ -0,0 +1,191 @@
+      }
+    })()
+  }, [
+    chainId,
+    asset,
+    chainAdapterManager,
+    contractAddress,
+    feeAsset.precision,
+    feeMarketData.price,
+    walletState.wallet,
+    dispatch,
+  ])
+
+  if (!state || !dispatch || !asset) return null
+
+  const handleConfirm = async () => {
+    if (!walletState.wallet || !contractAddress || !state?.userAddress || !dispatch) return
+    dispatch({ type: CosmosClaimActionType.SET_LOADING, payload: true })
+
+    const { gasLimit, gasPrice } = await getFormFees(asset, feeMarketData.price)
+
+    try {
+      const broadcastTxId = await handleStakingAction({
+        asset,
+        validator: contractAddress,
+        chainSpecific: {
+          gas: gasLimit,
+          fee: bnOrZero(gasPrice).times(`1e+${asset?.precision}`).toString(),
+        },
+        value: bnOrZero(claimAmount).times(`1e+${asset.precision}`).toString(),
+        action: StakingAction.Claim,
+      })
+      dispatch({ type: CosmosClaimActionType.SET_TXID, payload: broadcastTxId ?? null })
+      onNext(DefiStep.Status)
+    } catch (error) {
+      moduleLogger.error(error, { fn: 'handleConfirm' }, 'handleConfirm error')
+      toast({
+        position: 'top-right',
+        description: translate('common.transactionFailedBody'),
+        title: translate('common.transactionFailed'),
+        status: 'error',
+      })
+    } finally {
+      dispatch({ type: CosmosClaimActionType.SET_LOADING, payload: false })
+    }
+  }
+
+  const handleBack = history.goBack
+
+  return (
+    <>
+      <Stack alignItems='center' justifyContent='center' py={8}>
+        <Text color='gray.500' translation='defi.modals.claim.claimAmount' />
+        <Stack direction='row' alignItems='center' justifyContent='center'>
+          <AssetIcon boxSize='10' src={asset.icon} />
+          <Amount.Crypto
+            fontSize='3xl'
+            fontWeight='medium'
+            value={bnOrZero(claimAmount).div(`1e+${asset.precision}`).toString()}
+            symbol={asset?.symbol}
+          />
+        </Stack>
+      </Stack>
+      <Stack width='full' spacing={6}>
+        <Row>
+          <Row.Label>
+            <Text translation='defi.modals.claim.claimToAddress' />
+          </Row.Label>
+          <Row.Value>
+            <Skeleton minWidth='100px' isLoaded={!!state.userAddress}>
+              <Link
+                isExternal
+                color='blue.500'
+                href={`${asset?.explorerAddressLink}${state.userAddress}`}
+              >
+                {state.userAddress && <MiddleEllipsis value={state.userAddress} />}
+              </Link>
+            </Skeleton>
+          </Row.Value>
+        </Row>
+        <Row>
+          <Row.Label>
+            <Text translation='common.estimatedGas' />
+          </Row.Label>
+          <Row.Value>
+            <SkeletonText
+              noOfLines={2}
+              isLoaded={!!bnOrZero(state.claim.estimatedGasCrypto).gt(0)}
```

Comment:
`gt()` already returns a boolean, no need to cast as truthy falsy.

```suggestion
              isLoaded={bnOrZero(state.claim.estimatedGasCrypto).gt(0)}
```